Sources/OAuthSwiftClient.swift

```diff
@@ -30,19 +30,19 @@ open class OAuthSwiftClient: NSObject {
     public init(credential: OAuthSwiftCredential) {
         self.credential = credential
     }
-
-    public convenience init(consumerKey: String, consumerSecret: String, version: OAuthSwiftCredential.Version = .oauth1) {
-        let credential = OAuthSwiftCredential(consumerKey: consumerKey, consumerSecret: consumerSecret)
-        credential.version = version
+    
```

Comment:
this is a public init, so a breaking change
to maintain public api we can keep the old init, and call the new one. then add the deprecated annotation if the old one is no more necessary